tests/network/vmi_multus.go

```diff
@@ -1251,6 +1231,20 @@ var _ = SIGDescribe("[Serial]Macvtap", func() {
 	})
 })
 
+func expectInterfaceToExistByMac(virtClient kubecli.KubevirtClient, vmi *v1.VirtualMachineInstance, macAddress string, timeout int, expectError string) {
```

Comment:
The name `expectError` is confusing.
How about `assertMessage` ?

Reply:
Done